%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Graduate Curriculum Vitae
% LaTeX Template
% Version 1.1 (9/12/12)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Rensselaer Polytechnic Institute (http://www.rpi.edu/dept/arc/training/latex/resumes/)
%
% Important note:
% This template requires the res.cls file to be in the same directory as the
% .tex file. The res.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[margin, 10pt]{res} % Use the res.cls style, the font size can be changed to 11pt or 12pt here

\usepackage{helvet} % Default font is the helvetica postscript font
%\usepackage{newcent} % To change the default font to the new century schoolbook postscript font uncomment this line and comment the one above

\setlength{\textwidth}{5.1in} % Text width of the document

\newsectionwidth{1.5in} % How much horizontal space titles have

\headheight = -10pt % Move up top of page 10pts

\pagestyle{plain} % Page style with page numbers at center-bottom of every page

\begin{document}

%----------------------------------------------------------------------------------------
%	NAME AND CONTACT INFORMATION SECTION
%----------------------------------------------------------------------------------------

\moveleft.5\hoffset\centerline{\Large\bf Nathan LaPierre} % Your name at the top
\moveleft.5\hoffset\centerline{nathanrlapierre.com}
\moveleft.5\hoffset\centerline{NathanL2012@gmail.com}
%\moveleft.5\hoffset\centerline{(571) 839-5008}
 
\moveleft\hoffset\vbox{\hrule width\resumewidth height 1pt}\smallskip % Horizontal line after name; adjust line thickness by changing the '1pt'

%----------------------------------------------------------------------------------------

\begin{resume}

%----------------------------------------------------------------------------------------
%	POSITIONS & EMPLOYMENT SECTION
%----------------------------------------------------------------------------------------

\section{RECENT POSITIONS}

{\sl Postdoctoral Scholar} \hfill Aug. 2022 - Present \\ Department of Human Genetics, University of Chicago \\ Labs of Xin He and Matthew Stephens

{\sl Graduate Student Researcher} \hfill Sep. 2016 - June 2022 \\ Department of Computer Science, University of California, Los Angeles \\ Labs of Eleazar Eskin and Wei Wang


%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\section{EDUCATION}

PhD in Computer Science, June 2022 \\
University of California, Los Angeles

M.S. in Computer Science, 3.74 GPA, March 2019 \\
University of California, Los Angeles

B.S. in Applied Computer Science, 3.86 GPA ({\sl magna cum laude}), December 2015 \\
George Mason University
  
%----------------------------------------------------------------------------------------
%	PUBLICATIONS SECTION
%----------------------------------------------------------------------------------------

\section{PUBLICATIONS}

\bibliography{references} 
\bibliographystyle{ieeetr}

%Mohammad Arifur Rahman, \textbf{Nathan LaPierre}, Huzefa Rangwala, and Daniel Barbara, ``Clustering Metagenome Sequences Using Canopies", in {\sl 9th International Conference on Bioinformatics and Computational Biology}, Honolulu, Hawaii, 2017.

C. Cinelli, \textbf{N. LaPierre}, B. L. Hill, S. Sankararaman, and E Eskin, ''Robust Mendelian randomization in the presence of residual population stratification, batch effects and horizontal pleiotropy,'' \emph{Nature Communications}, vol. 13, no 1., pp. 1093, Mar. 2022.

\textbf{N. LaPierre}*, K. Taraszka*, H. Huang, R. He, F. Hormozdiari, and E. Eskin, ``Identifying Causal Variants by Fine Mapping Across Multiple Studies,'' \emph{PLOS Genetics}, vol. 17, no. 9, pp. e1009733, Sept. 2021.

F. Meyer et al (many authors), ``Critical Assessment of Metagenome Interpretation-the second round of challenges,'' \emph{bioRxiv}, July 2021.

J. Bloom et al (many authors), ``Massively scaled-up testing for SARS-CoV-2 RNA via next-generation sequencing of pooled and barcoded nasal and saliva samples,'' \emph{Nature Biomedical Engineering}, vol. 5, no. 7, pp. 657-665, Sep. 2020.

\textbf{N. LaPierre}, M. Alser, E. Eskin, D. Koslicki*, and S. Mangul*, ``Metalign: Efficient alignment-based metagenomic profiling via containment min hash,'' \emph{Genome Biology}, vol. 21, pp. e242, Sep. 2020.

%\textbf{N. LaPierre}*, K. Collins*, J. Rotman, and E. Eskin, ``Identifying Causal Variants by Fine Mapping Across Multiple Studies,'' in {\sl International Conference on Research in Computational Molecular Biology (RECOMB)}, virtual, June 2020, pp. 257-258.

\textbf{N. LaPierre}, R. Egan, W. Wang, and Z. Wang, ``De novo Nanopore read quality improvement using deep learning,'' \emph{BMC Bioinformatics}, vol. 20, no. 1, pp. e552, Dec. 2019.

\textbf{N. LaPierre}, C. Ju, G. Zhou, and W. Wang, ``MetaPheno: A Critical Evaluation of Deep Learning and Machine Learning in Metagenome-Based Disease Prediction,'' \emph{Methods}, vol. 166, pp. 74-82, Aug. 2019.

\textbf{N. LaPierre}*, S. Mangul*, M. Alser, I. Mandric, N.C. Wu, D. Koslicki, and E. Eskin, ``MiCoP: Microbial Community Profiling method capable of detecting low abundance viral and fungal organisms in metagenomic samples,'' \emph{BMC Genomics}, vol. 20, no. 5, pp. e423, June 2019.

M. A. Rahman, \textbf{N. LaPierre}, H. Rangwala, and D. Barbara, ``Metagenome sequence clustering with hash-based canopies,'' \emph{Journal of bioinformatics and computational biology}, vol. 15, no. 6, pp.1740006, Oct. 2017.

M. A. Rahman, \textbf{N. LaPierre}, and H. Rangwala, ``Phenotype Prediction from Metagenomic Data Using Clustering and Assembly with Multiple Instance Learning (CAMIL),'' \emph{IEEE/ACM transactions on computational biology and bioinformatics}, Oct. 2017. 

\textbf{N. LaPierre}, M. A. Rahman, and H. Rangwala, ``CAMIL: Clustering and Assembly with Multiple Instance Learning for Phenotype Prediction,'' in {\sl IEEE International Conference on Bioinformatics and Biomedicine}, Shenzhen, China, 2016.

\textbf{N. LaPierre} and H. Rangwala, ``Predicting Clinical Phenotype using OTU-based Metagenome Representation,'' in {\sl IEEE International Conference on Data Mining Workshop on Biological Data Mining and its Applications in Healthcare}, Atlantic City, New Jersey, 2015, pp. 156-163.

(*  Authors contributed equally)

%----------------------------------------------------------------------------------------
%	RESEARCH PRESENTATIONS SECTION
%----------------------------------------------------------------------------------------

\section{PRESENTATIONS}

``Accounting for Isoform Expression in eQTL Mapping,'' in {\sl CSHL Genome Informatics}, virtual, Nov. 2021.

``Metalign: Efficient alignment-based metagenomic profiling via containment min hash,'' in {\sl Intelligent Systems for Molecular Biology (ISMB) HitSeq}, virtual, July 2020.

``Identifying Causal Variants by Fine Mapping Across Multiple Studies'' in International Conference on Research in Computational Molecular Biology (RECOMB), June 2020.

``Metalign: Efficient alignment-based metagenomic profiling via containment min hash,'' in {\sl RECOMB-Seq}, virtual, June 2020.

``Identifying Causal Variants by Fine Mapping Across Multiple Studies'' in American Society for Human Genetics Annual Meeting, October 2019.

``MiniScrub: de novo long read scrubbing using approximate alignment and deep learning'' in Amazon Web Services - UCLA Computational Medicine Symposium, February 2019.

``MiniScrub: de novo long read scrubbing using approximate alignment and deep learning'' in American Society for Human Genetics Annual Meeting, October 2018.

``MiCoP: Microbial Community Profiling method capable of detecting low abundance viral and fungal organisms in metagenomic samples'' in American Society for Human Genetics Annual Meeting, October 2017.

``CAMIL: Clustering and Assembly with Multiple Instance Learning for Phenotype Prediction'' in IEEE International Conference on Bioinformatics and Biomedicine, December 2016.
\begin{itemize} \itemsep -2pt % Reduce space between items
%\item Conducted an oral presentation of the paper that I co-authored (as first author)
%\item Paper was published in the conference proceedings
\item Won a conference travel grant sponsored by NSF
\end{itemize}

``Predicting Clinical Phenotype using OTU-based Metagenome Representation'' in IEEE International Conference on Data Mining workshop on Biological Data Mining and its Applications in Healthcare, November 14, 2015
\begin{itemize} \itemsep -2pt % Reduce space between items
%\item Conducted an oral presentation of the paper that I co-authored (as first author)
%\item Paper was published in the conference workshop proceedings
\item Won a travel grant from the Undergraduate Student Travel Fund of the Office of Student Scholarship, Creative Activities, and Research at GMU
\end{itemize}

``Developing a Computational Pipeline for Metagenomic State Classification with Feature Engineering'' in Volgenau School of Engineering Undergraduate Research Celebration, April 2015
\begin{itemize} \itemsep -2pt % Reduce space between items
%\item Presented an earlier iteration of my research project in poster form
\item Won Outstanding Undergraduate Research Project Award for poster
\end{itemize}

%----------------------------------------------------------------------------------------
%	RESEARCH EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\section{RESEARCH EXPERIENCE}

{\sl PhD Student} \\
University of California, Los Angeles \hfill September 2016 - Present
\begin{itemize} \itemsep -2pt % Reduce space between items
\item Working with Professors Wei Wang, Eleazar Eskin, and Harold Pimentel
\item Developed a method for identifying causal variants (statistical fine mapping) by leveraging information from multiple studies using a Bayesian approach 
\item Developing multiple Mendelian Randomization methods providing sensitivity analysis and robustness to confounding from population structure
\item Contributed to the effort to develop \& deploy a saliva-based COVID-19 test (``Swab-Seq'') that is now deployed campus-wide at UCLA
\item Developing a method for isoform-aware eQTL mapping
\item Developed multiple published methods that perform accurate abundance profiling of microbial organisms based on metagenomic reads
\item Developed a method that uses deep learning to improve long sequencing read quality, in collaboration with scientists from the Joint Genome Institute
\item Two-time teaching assistant (courses: Computational Genetics and Machine Learning in Genetics), given an average rating of 8 out of 9 by my students
\item Mentored four undergraduate summer students, two of whom are now PhD students
\end{itemize}

{\sl Graduate Research Assistant (GRA) / Student Researcher} \\
George Mason University \hfill January - August 2016 (June-August as GRA)
\begin{itemize} \itemsep -2pt % Reduce space between items
\item Worked with Professor Huzefa Rangwala 
\item Developed CAMIL, a pipeline that uses multiple instance learning techniques based on whole metagenome shotgun sequence reads to predict whether or not a patient has a disease.
\item CAMIL paper accepted into IEEE BIBM 2016 (19\% acceptance rate).
\item Second author of paper on using canopy clustering and locality sensitive hashing to reduce clustering time for biological datasets.
\end{itemize}

{\sl Predicting Clinical Phenotype using OTU-based Metagenome Representation} \\
George Mason University  \hfill January 2015 - November 2015
\begin{itemize} \itemsep -2pt % Reduce space between items
\item Worked with Professor Huzefa Rangwala
\item Developed a computational pipeline that uses clustering and classification methods to quickly and accurately predict whether a patient has a disease based on a case/control metagenomic dataset
\item Paper accepted for publication, and poster presentation won Outstanding Undergraduate Project Award (see publications and presentations sections)
\end{itemize}

%----------------------------------------------------------------------------------------
%	PROFESSIONAL EXPERIENCE SECTION
%----------------------------------------------------------------------------------------
 
\section{WORK EXPERIENCE}

{\sl Graduate Student Researcher} \hfill Summer 2017 \\ DOE Joint Genome Institute / Lawrence Berkelely National Lab

\begin{itemize} \itemsep -2pt % Reduce space between items
\item Developed a method that uses a Convolutional Neural Network (Deep learning method) to improve Oxford Nanopore long read quality \emph{de novo}
\item Method improves both read accuracy and downstream \emph{de novo} assembly
\item Helped pioneer use of deep learning within the lab
\end{itemize}


{\sl Security Engineering Intern} \hfill Summers 2013-2015 \\
Sony Corporation of America 

\begin{itemize} \itemsep -2pt % Reduce space between items
\item Wrote secure and scalable software and worked with Big Data in order to help analyze, detect, and prevent attacks on Sony\textsc{\char13}s networks
\item Used Python, Javascript (Node, Express, Meteor), MongoDB, X/HTML, CSS 
\item Developed a workplace communications system using Meteor.js; solo project
\item Assisted in the development of a web application based on a searchable database system using Node.js, Express.js, and MongoDB
\item Assisted in the development of a network forensics system; wrote backend python scripts and XML web layouts
\end{itemize} 
 
%----------------------------------------------------------------------------------------
%	TEACHING EXPERIENCE SECTION
%----------------------------------------------------------------------------------------
 
\section{TEACHING EXPERIENCE}

{\sl Undergraduate Teaching Assistant} \hfill Fall 2014 - Spring 2015 \\
CS 306 - Computer Law and Ethics, Computer Science Department, George Mason University
\begin{itemize} \itemsep -2pt % Reduce space between items
\item Received Outstanding Undergraduate Teaching Assistant Award 
\item Assisted students with legal research, writing, and oral communication
\item Responsible for grading student assignments worth 25\% of their grade; one of the only Undergraduate Teaching Assistants entrusted with this responsibility
\end{itemize} 

%----------------------------------------------------------------------------------------
%	AWARDS and HONORS SECTION
%----------------------------------------------------------------------------------------

\section{AWARDS, FELLOWSHIPS, SCHOLARSHIPS} 

Selected Honors
\begin{itemize} \itemsep -2pt % Reduce space between items
\item {\sl Honorable Mention in NSF Graduate Research Fellowship}, 2015-16
\item {\sl Outstanding Undergraduate Student Award}, given to overall best undergraduate student in Computer Science at George Mason University, May 2016
\item {\sl Outstanding Academic Achievement Award}, given for outstanding performance in Computer Science, May 2016
\item {\sl Outstanding Undergraduate Research Project} for presentation of my research project at Volgenau School of Engineering Undergraduate Research Symposium, April 2015
\item {\sl Outstanding Undergraduate Teaching Assistant} for two semesters of excellence as a teaching assistant, April 2015
\item {\sl Dean's List} every semester
\end{itemize}

Fellowships and Merit-Based Scholarships
\begin{itemize} \itemsep -2pt % Reduce space between items
\item {\sl ModEling and uNdersTanding human behaviOR (MENTOR) NSF Training Grant}, 2018-19 academic year at UCLA
\item {\sl NIH T32 Doctoral Training Fellowship}, 2016-18 academic years at UCLA
\item {\sl Bersoff Endowed Scholarship}, Outstanding Academic Achievement, Awarded twice in 2015 and 2014
\item {\sl SWIFT Scholarship}, Outstanding Academic Achievement, Awarded in 2015
\end{itemize}

Honors Societies and Organizations
\begin{itemize} \itemsep -2pt % Reduce space between items
\item {\sl Honors College} at George Mason University
\item {\sl Alpha Lambda Delta Honor Society}
\item {\sl Golden Key International Honour Society}
\end{itemize}

Technical Competitions
\begin{itemize} \itemsep -2pt % Reduce space between items
\item {\sl Top 5 at VTHacks}, a software development competition at Virginia Tech with over 45 teams, April 2014
\item {\sl 2nd Place in the Technical Innovation Challenge}, a week-long competition at George Mason University to design a viable software product, jointly refereed by Computer Science and Business Departments, October 2014
\end{itemize}

%----------------------------------------------------------------------------------------
%	VOLUNTEER EXPERIENCE SECTION
%---------------------------------------------------------------------------------------- 

\section{VOLUNTEER EXPERIENCE}

Executive Curriculum Planner and Mentor, Community Programming Initiative \\ George Mason University Honors College and SRCT
 \hfill 2014 - 2015
\begin{itemize} \itemsep -2pt % Reduce space between items
\item Volunteer effort to teach basic programming to local 5th-8th grade students
\item Planned and developed parts of the curriculum for the sessions, such as designing games and hints to help the students create them
\item Mentored the elementary and middle school students during the sessions
\end{itemize} 

%----------------------------------------------------------------------------------------
%	COMPUTER SKILLS SECTION
%----------------------------------------------------------------------------------------

\section{COMPUTATIONAL \\ SKILLS} 

{\sl Languages / Scripting:} 
Python, R, C, Matlab/Octave, bash\\
{\sl Deep Learning Frameworks:} Keras, TensorFlow, PyTorch \\
{\sl Other Technologies:} Git, Docker, LaTeX \\
{\sl Operating Systems:} Linux, Windows, macOS \\
{\sl Experience with:} 
\begin{itemize} \itemsep -2pt % Reduce space between items
\item Machine Learning Algorithms: Deep Learning, Linear Regression, Logistic Regression, SVMs, Random Forests, etc
\item Machine Learning Applications: Clustering, Classification, Bioinformatics
\item Bioinformatics methods experience: statistical fine mapping, alignment, assembly, read error correction, metagenomics, abundance profiling
\item SGE and Slurm grid/cluster computing systems
\end{itemize}

%----------------------------------------------------------------------------------------
%	MENTORSHIP AND LEADERSHIP SECTION
%----------------------------------------------------------------------------------------

\section{MEMBERSHIPS AND LEADERSHIP}
{\sl Student-Run Computing and Technology (SRCT)} \hfill 2013 - 2016 \begin{itemize} \itemsep -2pt % Reduce space between items
\item Student organization at George Mason University that works on software projects and competitions to benefit the university and broader local community
\item Secretary and Member of Executive Board, Fall 2015 Semester
\end{itemize}

%----------------------------------------------------------------------------------------

\end{resume}
\end{document}